Makefile

```diff
@@ -84,7 +84,16 @@ SHELL = /usr/bin/env bash -o pipefail
 
 # Setting SED allows macos users to install GNU sed and use the latter
 # instead of the default BSD sed.
-SED ?= /usr/bin/sed
+SED := /usr/bin/sed
```

Comment:
I don't like having platform specific code.

Maybe we can have a simpler approach?

(in pseudocode)

```
SED=$(which gsed)
if SED is empty
  SED=$(which sed)
```

Reply:
+1
Also, using `command -v` would be better for following GNU specifications.

Reply:
I don't have any strong objections to using the `$SED --version`.